cmake/configure/configure_boost.cmake

```diff
@@ -47,6 +47,19 @@ MACRO(FEATURE_BOOST_CONFIGURE_BUNDLED)
     LIST(APPEND BOOST_DEFINITIONS "BOOST_ALL_NO_LIB")
     LIST(APPEND BOOST_USER_DEFINITIONS "BOOST_ALL_NO_LIB")
   ENDIF()
+
+  #
+  # Our bundled boost version 1.62 checks for the availability of
+  # "emplace_hint" incorrectly. Thus, simply define
+  # BOOST_NO_CXX11_HDR_UNORDERED_MAP if the gcc compiler version is less
+  # than 4.8.
+  #
+  IF( CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND
+      CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.8" )
```

Comment:
Yes. I went through the original bug reports again. It seems to work with 4.8, but not with 4.7.